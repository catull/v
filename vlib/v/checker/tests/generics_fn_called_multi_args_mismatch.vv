fn main() {
	x := 'ab'.runes()[..1]
	foo_str(1, x)
}

fn foo_str<T>(b T, a string) {
}
