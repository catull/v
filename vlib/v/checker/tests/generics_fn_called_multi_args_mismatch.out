vlib/v/checker/tests/generics_fn_called_multi_args_mismatch.vv:3:13: error: cannot use `[]rune` as `string` in argument 2 to `foo_str`
    1 | fn main() {
    2 |     x := 'ab'.runes()[..1]
    3 |     foo_str(1, x)
      |                ^
    4 | }
    5 |
